storybook/src/components/Accordion/Accordion.test.stories.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import type { Meta, StoryObj } from '@storybook/react-vite'
+
+import { Accordion } from '@amsterdam/design-system-react/src'
+
+import { getVariants } from '../../utils/getVariants'
+import { default as accordionMeta } from './Accordion.stories'
+
+const meta = {
+  ...accordionMeta,
+  title: 'Components/Containers/Accordion',
+} satisfies Meta<typeof Accordion>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Test: Story = {
+  render: (args) =>
+    getVariants({
+      component: Accordion,
+      args,
+      children: (
+        <>
```

Comment:
We use arrays for multiple children because a React Fragment renders less intuitively in Code View. That’s no problem here, but it’s still useful to follow one approach. Multiple occurrences in this PR.